ddp-workspace/projects/toolkit/src/lib/components/dialogs/sessionWillExpire.component.ts

```diff
@@ -68,13 +72,28 @@ export class SessionWillExpireComponent implements OnInit, OnDestroy {
 
     public signOut(): void {
         this.renewNotifier.hideSessionExpirationNotifications();
-        this.session.disableTokenExpiredProcess();
         this.auth0.logout();
     }
 
     public renewSession(): void {
-        this.renewNotifier.hideSessionExpirationNotifications();
-        this.session.disableTokenExpiredProcess();
+        // Blocks UI to prevent interference in the session renewing process
+        this.isRenewing = true;
+        this.dialogRef.disableClose = true;
```

Comment:
Nice! I think I noticed one time that I was able to dismiss the dialog. Glad to see you have added the safeguard here!